src/install/index.ts

```
@@ -74,6 +83,103 @@ async function promptDockerChoice(): Promise<boolean> {
+    if (!exists) {
+      missingCommands.push(cmd);
+    }
+  }
+
+  if (missingCommands.length === 0) {
+    logSuccess('Rootless prerequisites detected');
+    return;
+  }
+
+  logInfo('Missing rootless prerequisites:');
+  missingCommands.forEach((cmd) => logInfo(`  • ${cmd}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)